practices/oocss.md

```diff
@@ -1,12 +1,39 @@
 # Object Oriented CSS
 
+
+
 ## What is OOCSS
 
+OOCSS or 'Object Orientated CSS' is an approach to writing css that aims to encourage reusable, maintainable and predictable css. It has spawned a number of different approaches to standardise the concept most of which adhere to the core principles of OOCSS outline below.
+
+A CSS 'object' is a repeating visual pattern, that can be abstracted into an independent snippet of HTML, CSS, and possibly JavaScript. That object can then be reused throughout a project. The aim is to break down the css declaration into the smallest blocks possible in order to maximise their potential for reuse.
+
+An important component of OOCSS is the process of defining repeating visual features as separate modifying 'skins' that you can apply as distinctive classes to allow visual variety regardless of the structure of the HTML. In order to do this css declarations should not rely upon any assumptions about the semantics of the HTML. Instead classes are used to define 'objects' and the components therein. The layout and display of particular object or components can then be modified by additional classes or 'skins'.
```

Comment:
"An important component of OOCSS is the process of defining repeating visual features as separate modifying 'skins' that you can apply as distinctive classes to allow visual variety regardless of the structure of the HTML." < is it possible to somehow break this sentence down?